### PR TITLE
Modify default spec template

### PR DESCRIPTION
**Pull request by kasicka**

### CODE REVIEW

File: default.n2r [hunk 1]
```diff
@@ -4,34 +4,30 @@
...
+%global enable_tests 0
+
+%{?nodejs_find_provides_and_requires}
+
 Name: nodejs-%{npm_name}
 Version: $VERSION
 Release: $RELEASE%{?dist}
 Summary: $SUMMARY
 License: $LICENSE
-Group: Development/Libraries
```
Comment (kasicka) [inline default.n2r:12]: This is deprecated.

File: default.n2r [hunk 2]
```diff
@@ -4,34 +4,30 @@
...
 URL: $PROJECTURL
 $SOURCES
-Requires: nodejs(engine)
-BuildRequires: nodejs-devel
 BuildRequires: nodejs-packaging
-BuildRequires: npm
```
Comment (kasicka) [inline default.n2r:18]: nodejs-packaging should take care of basic dependencies.

File: default.n2r [hunk 3]
```diff
@@ -4,34 +4,30 @@
...
-BuildArch: noarch
+BuildArch:  noarch
+
+%if 0%{?fedora} >= 19
 ExclusiveArch: %{nodejs_arches} noarch
-Provides: npm(%{npm_name}) = %{version}
+%else
+ExclusiveArch: %{ix86} x86_64 %{arm} noarch
+%endif
```
Comment (kasicka) [inline default.n2r:25]: Older systems might not support %{nodejs_arches}, so this is safer.

File: default.n2r [hunk 4]
```diff
@@ -4,34 +4,30 @@
...
-Requires: nodejs(engine)
-BuildRequires: nodejs-devel
 BuildRequires: nodejs-packaging
-BuildRequires: npm
-BuildArch: noarch
+BuildArch:  noarch
+
+%if 0%{?fedora} >= 19
 ExclusiveArch: %{nodejs_arches} noarch
-Provides: npm(%{npm_name}) = %{version}
```
Comment (kasicka) [inline default.n2r:21]: Also should be taken care of by nodejs-packaging.

File: default.n2r [hunk 5]
```diff
@@ -4,34 +4,30 @@
...
+%else
+ExclusiveArch: %{ix86} x86_64 %{arm} noarch
+%endif
+
 $PROVIDES
 
 %description
-$DESCRIPTION
-
-%package doc
```
Comment (dLobatog) [inline default.n2r:27]: Not sure why you remove the doc section?

**Comment by dLobatog** [comment]
@kasicka :+1: to everything but removing the doc section - why? 

Thanks for the PR - it's great to see this being used outside of https://github.com/theforeman/foreman-packaging/ which was the main motivation, what do you use it for (if you can share it)

File: default.n2r (hunk 5 above)
Comment (kasicka) [reply default.n2r:27]: Only large or multi-file documentation should go to separate doc subpackage, which is very rare in nodejs modules and I can't remember seeing it anywhere except npm.

**Comment by kasicka** [comment]
I'd like to use it for Fedora/EPEL and software collections packaging.

**Comment by dLobatog** [comment]
Fair enough, I guess I'll create our own template for foreman-packaging and leave this as a general one. Thanks @kasicka ! Let me know if you need any more fixes or help with packaging for Fedora/EPEL, I'd be glad to help (newbie though :smile:)

**Pull request merged**